crates/ruff/resources/test/fixtures/pylint/invalid_envvar_value.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+os.getenv(1) # [invalid-envvar-value]
+os.getenv("a")
+os.getenv('test')
+
+os.getenv(["hello"]) # [invalid-envvar-value]
+
+AA = "aa"
+os.getenv(AA)
```

Comment:
Nit: can you add a newline here? Or better yet, run this file through Black?

Reply:
Fixed :)